src/lib/docs/Antelope/Time.md

```diff
@@ -3,18 +3,17 @@ title: Time
 description: change_me
 category: Antelope
 published: true
-requiresReview: true
 ---
 
 # Time
 
-The `Time` type represents the date format used by the blockchain and contains a number of helpers to convert between Javascript Dates objects and Antelope Time formats.
+The `Time` type represents the date format used by the blockchain and contains a number of helpers to convert between Javascript Date objects and Antelope Time formats.
 
 ## Variants
 
-The `Time` set of types has 3 variants, all used at various points within the Antelope stack:
+The `Time` type has 3 variants, all used at various points within the Antelope stack:
```

Comment:
Not sure about this one, but there may be a semantic shift here too.

Reply:
fixed